packages/models-library/src/models_library/function_service/demo_units.py

```diff
@@ -27,141 +27,157 @@ def build_input(schema):
                 {
-                    "title": "Time",
-                    "description": "Positive time base unit",
+                    "label": "Time",
+                    "description": "Positive time in microseconds",
                     "minimum": 0,
-                    "x_unit": "s",
+                    "x_unit": "micro-second",
                     "type": "number",
                 }
             ),
             "current": build_input(
                 {
-                    "title": "Current",
+                    "label": "Current",
                     "description": "Current base unit",
-                    "x_unit": "A",
+                    "x_unit": "ampere",
                     "type": "number",
                 }
             ),
             "luminosity": build_input(
                 {
-                    "title": "Luminosity",
+                    "label": "Luminosity",
                     "description": "Luminosity base unit",
-                    "x_unit": "cd",
+                    "x_unit": "candela",
                     "type": "number",
                 }
             ),
             "mass": build_input(
                 {
-                    "title": "Mass",
-                    "description": "Positive mass base unit",
+                    "label": "Mass",
+                    "description": "Positive mass in micrograms",
                     "minimum": 0,
-                    "x_unit": "g",
+                    "x_unit": "micro-gram",
                     "type": "number",
                 }
             ),
             "substance": build_input(
                 {
-                    "title": "Substance",
-                    "description": "Substance base unit",
+                    "label": "Substance",
+                    "description": "Substance base in millimoles",
                     "minimum": 0,
-                    "x_unit": "mol",
+                    "x_unit": "milli-mole",
                     "type": "number",
                 }
             ),
             "temperature": build_input(
                 {
-                    "title": "Temperature",
+                    "label": "Temperature",
                     "description": "Temperature in Kelvin",
                     "minimum": 0,
-                    "x_unit": "K",
+                    "x_unit": "kelvin",
                     "type": "number",
                 }
             ),
             "angle": build_input(
                 {
-                    "title": "Angle",
+                    "label": "Angle",
                     "description": "Angle in degrees",
-                    "x_unit": "deg",
+                    "x_unit": "degree",
+                    "type": "number",
+                }
+            ),
+            "velocity": build_input(
+                {
+                    "label": "Velo-city",
+                    "description": "Velocity in meters per second",
+                    "x_unit": "meter_per_second",
```

Comment:
why this format for ``x_unit``? Is it part of some standard?

Aha ... i see you took it direclty from https://github.com/hgrecco/pint/blob/master/pint/default_en.txt

MMhhh i wonder what will we do when we have to represent something like acceleration (``m/s2``) or something more elaborated as``m2 kg s-2 K-1``... I guess we need to explore the library a bit more for that.